idb/postgres/internal/encoding/types.go

```diff
@@ -1,24 +1,111 @@
 package encoding
 
 import (
-	sdk_types "github.com/algorand/go-algorand-sdk/types"
-
-	"github.com/algorand/indexer/types"
+	"github.com/algorand/go-algorand/crypto"
+	"github.com/algorand/go-algorand/data/basics"
+	"github.com/algorand/go-algorand/data/bookkeeping"
+	"github.com/algorand/go-algorand/data/transactions"
 )
 
+type blockHeader struct {
```

Comment:
I think the current way is better for readability because you can clearly see how the types are composed and how encoding/decoding functions call each other. For reviewing, the types in all three files are ordered the same, so you can do a linear scan on 3 tabs (4 tabs if you also want to review `encoding_test.go`).